LICENSE

```diff
@@ -73,7 +73,7 @@ analyze, test, perform and/or display publicly, prepare derivative works,
 distribute, and otherwise use Python alone or in any derivative version,
 provided, however, that PSF's License Agreement and PSF's notice of copyright,
 i.e., "Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010,
-2011, 2012, 2013, 2014, 2015, 2016, 2017 Python Software Foundation; All Rights
+2011, 2012, 2013, 2014, 2015, 2016, 2017 2018 Python Software Foundation; All Rights
```

Comment:
Missing a comma after 2017 and it looks like "Rights" should flow to the next line to maintain consistent wrapping.